### DISCUSSION TITLE
Pessoal, posso divulgar uma vaga para programador aqui?

### DISCUSSION THREAD

**Discussion by Milka0102** [community/community#134894] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Pessoal, não sou da área e estou me sentindo meio perdida aqui! É possível divulgar uma vaga para Desenvolvedor(a) Júnior - Front End - Remoto aqui?

**Comment by ebndev** [comment]
Hi @Milka0102, thanks for being a part of the GitHub Community!

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
